docs/usecase.rst

```diff
@@ -21,12 +21,47 @@ Scripting user
 
 Python code written directly against the data flow client library.
 
+.. seealso:: :doc:`dataflow`
+
+Run a sequence of dependent tasks
+---------------------------------
+
+| 1. Import data flow client package.
+| 2. Add an operation to the work graph.
+| 3. Add a dependent operation to the work graph.
+| 4. Request results.
+|  4.1 Work is implicitly dispatched.
+|  4.2 Interpreter blocks until results are available.
+|  4.3 API data is localized and converted to native data types.
```

Comment:
Here 'native data types' refers to the data types of the runtime system?

Reply:
This is a client use case. "Native data types" refers to the client environment, such as a Python interpreter (running on a machine outside of the context of the computing resource allocation, in the case of dispatched execution).

Reply:
Then I am not sure it this is within the scope of the runtime system? Without 'dispatched execution' do you need a runtime system?